ApolloExtra.lua

```diff
@@ -3260,103 +3351,53 @@ OlympusLootData.ApolloUpgrade = {
 			{
                 Name = "ApolloMakeUp01",
 				PreEventFunctionName = "BoonInteractPresentation", PreEventFunctionArgs = { PickupWait = 1.0, },
-				{ Cue = "/VO/Apollo_0069",
+				{ Cue = "/VO/Apollo_0110",
 					Emote = "PortraitEmoteAffection",
 					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
-					Text = "Let's say we kissed and made up, dearest. Just, do not anger me again, or I might come down and kill you there myself!" },
+					Text = "I sense you've came back to the light, Zagzag. Sorry, I just had to make sure." },
 			},
 			ApolloMakeUp02 =
 			{
                 Name = "ApolloMakeUp02",
 				PreEventFunctionName = "BoonInteractPresentation", PreEventFunctionArgs = { PickupWait = 1.0, },
-				{ Cue = "/VO/Apollo_0070",
+				{ Cue = "/VO/Apollo_0111",
 					Emote = "PortraitEmoteCheerful",
 					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
-					Text = "Alas that passion sometimes drives me into vengeance, dearest! Yet, who among us isn't without vices, {#DialogueItalicFormat}hm{#PreviousFormat}?" },
+					Text = "I knew this moment would come. No hard feeling, Zagzag." },
 			},
 			ApolloMakeUp03 =
 			{
                 Name = "ApolloMakeUp03",
 				PreEventFunctionName = "BoonInteractPresentation", PreEventFunctionArgs = { PickupWait = 1.0, },
-				{ Cue = "/VO/Apollo_0071",
+				{ Cue = "/VO/Apollo_0112",
 					Emote = "PortraitEmoteAffection",
 					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
-					Text = "Oh, how I hate to fight with you like this! Let's forgive each other and forget, go back to how things used to be?" },
+					Text = "Why are we fighting again, Zagzag? You know what I forgive you. Just don't do it again." },
```

Comment:
"Why are we fighting again, Zagzag? You know what? I forgive you. Just don't do it again." },